articles/libraries/machine-learning/basic-classification.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Quantum machine learning library
+description: Learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the Microsoft QDK.
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/16/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.basics
+---
+
+# Basic classification: Classify data with the QDK
+
+In this Quickstart, you will learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK. 
+
+In this guide we will use the half-moon dataset, using a classifier structure defined in Q#.
+
+## Execute the classifier from the host program
+
+First, let's take a look at the host programs to see what the process looks like. Your host program will consist of three parts:
+
+- Loading of the data from the dataset.
+- Execution of the training to obtain the parameters and the bias that determine the model.
+- Validation of the model to test the accuracy.
+
+    ### [Python with Visual Studio Code or the Command Line](#tab/tabid-python)
+
+    To run your the Q# classifier from Python, save the following code as `host.py`. Remember that you also need the Q# file `Training.qs` that is explained later in this tutorial.
+
+    :::code language="python" source="~/quantum/samples/machine-learning/half-moons/host.py" range="3-42":::
+
+    You can then run your Python host program from the command line:
+
+    ```bash
```

Comment:
It wasn't clear to me whether you could run the code at this point or not

Reply:
If you can't run it, then we should not give instructions to do that

Reply:
Brief explanation of the starting parameters would be good

Reply:
Yes, true. This was initially written for the Wine sample that didn't have them, but we had to go for Half-moons. Let me add a brief comment.

Reply:
You can run it provided the `Training.qs` file. This file is explained later because it is considerably more complex and daunting than the host programs. I thought it was a good a idea to present the general structure of the process with the host program to have an easier understanding of 'Training.qs' later.